app/controllers/api/notes_controller.rb

```diff
@@ -294,11 +294,25 @@ def search
           raise OSM::APIBadUserInput, "Date #{params[:to]} is in a wrong format"
         end
 
-        @notes = @notes.where(:created_at => from..to)
+        @notes = params[:sort] == "created_at" ? @notes.where(:created_at => from..to) : @notes.where(:updated_at => from..to)
       end
 
       # Find the notes we want to return
```

Comment:
This comment should be moved down to stay with the line it was originally attached to as it's incorrect where it is now.

If you want a comment here then it should say `Chose the sort order` or something.